apps/fabric-website-resources/src/customizations/customizations.ts

```diff
@@ -1,12 +1,15 @@
 import { IExampleCardCustomizations, IAppCustomizations } from '@uifabric/example-app-base';
 import { FluentCustomizations } from '@uifabric/fluent-theme';
 import { DefaultCustomizations, TeamsCustomizations, WordCustomizations } from '@uifabric/theme-samples';
+import { AzureCustomizationsLight, AzureCustomizationsDark } from '@uifabric/azure-themes';
 
 const exampleCardCustomizations: IExampleCardCustomizations[] = [
   { title: 'Default', customizations: DefaultCustomizations },
   { title: 'Fluent', customizations: FluentCustomizations },
   { title: 'Word', customizations: WordCustomizations },
-  { title: 'Teams', customizations: TeamsCustomizations }
+  { title: 'Teams', customizations: TeamsCustomizations },
+  { title: 'AzureLight', customizations: AzureCustomizationsLight },
+  { title: 'AzureDark', customizations: AzureCustomizationsDark }
```

Comment:
Do you want these available in `experiments` package as well? If so, you can also edit [customizations there](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/experiments/src/demo/customizations.ts).